release-notes/v0.4.0.md

```diff
@@ -12,6 +12,7 @@
 #### Data types
 - Implement `AsciiString` for ASCII-only strings [[#377][377]]
 - Implement `Utf8String` for strings with non-ASCII content [[#385][385]]
+- Use more predictable format in `float` and `double` conversion to string [[#414][414]]
```

Comment:
TODO: Settle on a title for this change and use it consistently, both here and in the commit message. :innocent:

Reply:
Fixed, resolving. I realized now why this was inconsistent: the commit message was too long, causing GitHub to wrap it. (Should probably add a pre-commit check locally to prevent me from pushing commits which GitHub will wrap or something...)